script/README.md

```diff
@@ -12,20 +12,23 @@ To use these scripts, please follow these steps **first**:
 - [ ] If you're using GitHub Enterprise, and access to github.com is limited within your organization, import the following repos and all their branches in your newly created organization: [caption-this](https://github.com/githubtraining/caption-this), [polygons](https://github.com/githubtraining/polygons), [github-games](https://github.com/githubtraining/github-games). You'll need access to GitHub.com to perform this one-time operation.
   - _Note: For the [caption-this](https://github.com/githubtraining/caption-this) repository to function properly, the `baseurl` parameter in `_config.yml` needs to be the exact same as the GH pages link generated by GHE._
 - [ ] Ensure your user account is an owner of the newly created training org, and add any other trainers as owners of the org. Alternatively, you can create a separate teaching account that all trainers have access to, and make that teaching account an owner of the org.
-- [ ] Create a [Personal Access Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) for your user account, or the shared teaching account. You'll need this for the [local configuration](#local-configuration).
+- [ ] Create a [Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) for your user account, or the shared teaching account. You'll need this for the [local configuration](#local-configuration).
+  - _Note: Your personal access token must include the scopes **repo** and **delete_repo**._
```

Comment:
Shouldn't we also include it here that the PAT must be authorized to access the Organization where the training repositories will be created? I know it is very straightforward, but documenting it would make it more fool-proof 😃

Reply:
Good point, @EduSemensati. Adding it now.

Reply:
You added the scopes, but not a note about the need to authorize for SAML single sign-on: https://docs.github.com/en/github/authenticating-to-github/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on